eth/ethconfig/config.go

```diff
@@ -170,8 +171,9 @@ type Config struct {
 	Miner miner.Config
 
 	// Transaction pool options
-	TxPool   legacypool.Config
-	BlobPool blobpool.Config
+	TxPool     legacypool.Config
+	BlobPool   blobpool.Config
+	BundlePool bundlepool.Config
```

Comment:
The gen_config.go file needs to be regenerated

Reply:
fixed, please review again.